internal/sponge/sfix/sfix.go

```diff
@@ -0,0 +1,50 @@
+package sfix
+
+import (
+	"encoding/json"
+	"os"
+
+	"github.com/ardanlabs/kit/db"
+	"github.com/coralproject/shelf/internal/sponge"
+	"github.com/coralproject/shelf/internal/sponge/item"
+	mgo "gopkg.in/mgo.v2"
```

Comment:
goimports.  The point here is moot as sfix is now removed. I will resolve this before it comes around next time.